components/org.wso2.carbon.identity.inbound.ldap/src/main/java/org/wso2/carbon/identity/inbound/ldap/provider/LDAPConstants.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.inbound.ldap.provider;
+
+public class LDAPConstants {
+
+
+    public static final String LDAP_CLAIM_URI = "http://wso2.org/ldap/claim/";
+
+    public static class LDAPSchemaConstants {
+
+        public static final String LDAP_SCHEMA_URI = "http://wso2.org/ldap/claim";
+        public static final String GIVEN_NAME_URI = "http://wso2.org/ldap/claim/givenName";
+        public static final String GIVEN_NAME = "givenName";
```

Comment:
Since this is a new claim dialect for IS, we need to put detailed javadoc comments. This must need to go into userguide doc as well.

The userguide should say why you had to introduce a new claim dialect for ldap.

Reply:
Added Java doc comment.
The purpose is already defined in the user guide.